src/sagemaker/pipeline.py

```diff
@@ -49,6 +50,14 @@ def __init__(self, models, role, predictor_cls=None, name=None, vpc_config=None,
         self.sagemaker_session = sagemaker_session
         self._model_name = None
 
+    def enable_network_isolation(self):
+        """Whether to enable network isolation when creating this Model
+
+        Returns:
+            bool: If network isolation should be enabled or not.
+        """
+        return False
```

Comment:
I copied this from model.py but didn't really understand it -- is this just supposed to always be false?

Reply:
For now yes it should be false. Only Models that are consumed from Marketplace set this to True.

Reply:
Ah, ok